cli/wadl2dot/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Dot.scala

```diff
@@ -97,6 +97,9 @@ object Wadl2Dot {
   val help = parser.flag[Boolean] (List("h", "help"),
                                    "Display usage.")
 
+  val xpathVersion = parser.option[Int](List("t", "xpath-version"), "n",
```

Comment:
Although the Wadl2Checker doesn't support '-e/--show-errors' or '-n/--nfa-mode' (as it shouldn't), so just removing this one would still be consistent.
I just thought it was an oversight that it wasn't there.